pkg/dependencies/dependencies.go

```diff
@@ -0,0 +1,172 @@
+// Copyright 2023 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dependencies
```

Comment:
Nit: Not blocker

`dependencies.Dependency` is quite a long name, we might abbreviate one of the sides to `deps`

Reply:
🤔 why not call the pkg `dependency` in singular? it aims to represent a Dependency, some packages use this format and then dependency.Dependency as main struct exported

Reply:
I agree on going with `deps` for the package. I don't like having it in singular given that the dependencies are repositories and the name in the manifest is in plural

Reply:
Renamed to `deps` in 06889461e9b45ff07ee136d4b52f602098e3c688